### PR TITLE
GH-815: Bumped up the node-pty version.

### LINKED ISSUE #815: [windows] Spawned processes leak on Windows 10
Start a bundled electron application, kill the application, one can still see stale processes on Windows.

See: https://github.com/theia-ide/yangster-electron/issues/8
Also: https://github.com/Microsoft/vscode/issues/26807

### PR DESCRIPTION
**Pull request by kittaakos**
The new version is intentionally 0.7.1 to be sync with VSCode.
Closes #815.

Signed-off-by: Akos Kitta <kittaakos@gmail.com>

### CODE REVIEW

**Comment by hexa00** [comment]
So the only reason is to be in sync ? why do we want to be ?

File: packages/process/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "Theia process support.",
   "dependencies": {
     "@theia/core": "^0.2.0",
-    "node-pty": "^0.7.0"
+    "node-pty": "^0.7.1"
```
Comment (svenefftinge) [inline packages/process/package.json]: why not 0.7.2?

Comment (kittaakos) [reply packages/process/package.json]: They have a broken build. But this did not fix anything. So does not matter.

**Comment by kittaakos** [comment]
Did not fix anything. I am closing this.

**Comment by svenefftinge** [comment]
I think in general it is good to upgrade our deps to latest releases, since usually they include fixes.

**Comment by kittaakos** [comment]
> I think in general it is good to upgrade our deps to latest releases, since usually they include fixes.

I agree. So that's why I went to their CIs, checked the history and they had to trigger the builds three times to be able to release.

screen shot 2017-11-10 at 15 35 33

**Comment by hexa00** [comment]
See also #817 since RawProcess/TerminalProcess have more testing into the process exit etc... 

It may fix the issue.

**Comment by kittaakos** [comment]
> It may fix the issue.

Cool, thanks! Let me try. It is just complicated to bundle an application from the Theia source.

**Comment by Tyriar** [comment]
Test flakiness was caused by https://github.com/Tyriar/node-pty/issues/159 (0.7.3).

**Pull request closed without merging**